src/_posts/2018-06-22-beam-2.5.0.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title:  "Apache Beam 2.5.0"
+date:   2018-06-22 00:00:01 -0800
+excerpt_separator: <!--more-->
+categories: blog
+authors:
+  - aromanenko
+---
+
+We are glad to present the new 2.5.0 release of Beam. This release includes
+multiple fixes and new functionalities. <!--more--> For more information
+please check the detailed release notes.
+
+# New Features / Improvements
+
+## Go SDK support
+The Go SDK has been officially accepted into the project, after an incubation period and community effort. Go pipelines run on Dataflow and all portable runners, notably the Flink runner. More details are [here(https://beam.apache.org/documentation/sdks/go/)].
```

Comment:
Portable Flink runner isn't part of the release and not ready yet, so please remove this reference.

Reply:
Fixed